server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
I vote for re-org. If we get adopters other than us, we can worry about changing things like this and handle it with version numbering. For now though, we're in the unstable API mode.